city_scrapers/spiders/cook_pharmaceutical_disposal.py

```diff
@@ -0,0 +1,114 @@
+from datetime import datetime, time
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CookPharmaceuticalDisposalSpider(CityScrapersSpider):
+    name = "cook_pharmaceutical_disposal"
+    agency = "Cook County Pharmaceutical Disposal Advisory Committee"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.cookcountysheriff.org/rx/advisory-committee/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self.base_url = "https://www.cookcountysheriff.org"
+
+        # Link to current meeting shown as a single image
```

Comment:
I think we won't plan on looking into OCR here, so we can remove this comment block, good to know though, thanks!